templates/github/.ci/scripts/collect_changes.py.j2

```diff
@@ -1,15 +1,17 @@
+#!/bin/env python3
 {% include "header.j2" %}
 
 import itertools
 import os
 import re
+import tomllib
 
-import toml
```

Comment:
There's a place somewhere where we install this now uneeded library.

Reply:
I've removed it now